tests/fuzzing/fuzz_fpgainfo.sh

```diff
@@ -0,0 +1,265 @@
+#!/bin/bash
```

Comment:
Could you please rename this file to fuzz-fpgainfo.sh for consistency with the other scripts?

Reply:
rename fuzz_fpgainfo.sh to fuzz-fpgainfo.sh